mobile/src/main/java/com/alexstyl/specialdates/addevent/WriteableAccountsProvider.java

```diff
@@ -44,15 +44,16 @@ public WriteableAccountsProvider(AccountManager accountManager, PackageManager p
     }
 
     private static boolean accountIsWritable(String accountType) {
-        // TODO how do you even?
         return GOOGLE_ACCOUNT.equals(accountType);
     }
 
-    private static AuthenticatorDescription getAuthenticatorDescription(String type,
```

Comment:
There is no particular usage of Generics in here. I just simplified the for loop as the IDE suggested